source/index.html.slim

```diff
@@ -1,59 +1,66 @@
+- past_events = data.webmondays.map { |past| past if d(past.date) < d(today_date)}.compact
+- upcoming_events = data.webmondays.map { |later| later if d(later.date) >= d(today_date)}.compact
+- next_event = upcoming_events.pop
 #content
-  #content-en.content lang="en" 
+  #content-en.content lang="en"
     .event-box
       .event-box-inner
-        h3 Next event
-        - data.webmondays.each_with_index do |webmontag, index|
-        	- if webmontag.state == 'next'
-  		      .event-date 
-  		      	=webmontag.en.date
-  		      .event-location 
-  		      	=data.config.page.place.name
-  		      .event-address 
-  		      	="#{data.config.page.place.street}, #{data.config.page.place.number} #{data.config.page.place.city}"
-  		      .event-attend
-  		        p Attend now!
-  		      .event-attend-buttons
-  		        a.button href=webmontag.facebook_url
-  		          i.icon-facebook
-  		          = "Facebook"
-  		        a.button href=webmontag.wiki_url
-  		          i.icon-wikipedia
-  		          = "Wiki"
-  		      .event-request
-  		        p
-  		          = "Do you want to share something? "
-  		          <br/>
-  		          = "Add your talk to the "
-  		          a href=webmontag.wiki_url wiki
-  		          = " or "
-  		          a href=webmontag.facebook_url "comment it on facebook"
+          h3 Next event
+	      .event-date
+	      	=next_event.date
```

Comment:
that's a good idea :+1: 

in that case it would be good if you would create that ticket and mentioned it in this PR